cmd/main.go

```diff
@@ -86,9 +86,19 @@ func probeHandler(w http.ResponseWriter, r *http.Request, logger log.Logger, con
 	defer cancel()
 	r = r.WithContext(ctx)
 
+	module := r.URL.Query().Get("module")
+	if module == "" {
+		http.Error(w, "Module parameter is missing", http.StatusBadRequest)
+		return
```

Comment:
Rather than an error, we could make `module == ""` set `default`. Similar to how we have a default value in the blackbox and snmp exporters.

Reply:
Updated. Please take a look.